ClassLibrary1/Entities/Menu.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using WhatShouldITest.Interfaces;
+using WhatShouldITest.Utilities;
+
+namespace WhatShouldITest.Entities
+{
+    public class Menu
+    {
+        private readonly IMenuItemProvider _menuItemProvider;
+
+        public Menu(IMenuItemProvider menuItemProvider)
+        {
+            _menuItemProvider = menuItemProvider;
+            DailySpecial = MenuUtilities.GetDailySpecial(DateTime.Now.DayOfWeek);
```

Comment:
The test we've written is actually testing this line of code. The only thing that is happening in this line is an assignment. We're just testing that the .NET assignment operator is working. We don't need to test that. If that's not working, we are going to be having MUCH larger problems. We don't need to cover this line. There is only one path this code can take. There is no branching logic.

Some folks would say that we could rewrite the test to ensure that the CORRECT special is being stored, but this isn't the place for that. The predicate isn't here.